src/app/dashboard/users/page.tsx

```diff
@@ -210,12 +268,16 @@ async function DashboardUsersContent({ searchParams }: Props) {
   const period = getPeriod(searchParams, settings)
   const [usersData, totalDuration, usersCount, totalRequests] =
     await Promise.all([
-      getUsers(period.daysAgo, period.date, period.string),
+      getUsers(period.daysAgo, period.date, period.string, settings),
       getTotalDuration(period.string, settings),
       getUsersCount(settings),
       getTotalRequests(period.date, settings),
     ])
 
+  if (!usersData) {
+    return undefined
```

Comment:
This conditional will lead to nothing being rendered and no errors showing. There will always be at least 1 user, so I don't think it's necessary even.

<img width="1326" alt="image" src="https://github.com/user-attachments/assets/e78f8f46-7462-40a4-b9ad-f82953d136c3">